src/pages/informacija-lietuviams/{MarkdownRemark.frontmatter__slug}.js

```diff
@@ -31,10 +31,13 @@ export default function Template({ data, pageContext }) {
 
   return (
     <Layout pagePath="/apie-mus/">
-      {frontmatter && frontmatter.title_override ? (
-        <Title>{frontmatter.title_override}</Title>
+      {frontmatter && frontmatter.page_title ? (
```

Comment:
What happened to the title_override? I still see it in forestry page content-type content.

Reply:
`title_override` is still being used in breadcrumbs and in the FAQ buttons. Wanted to separate those, because otherwise the buttons would get bloated with that long of a text:
![image](https://user-images.githubusercontent.com/69549795/202704517-7b698a88-1b32-48b2-86b9-54f1e0a3c8c6.png)

Throughout the whole PR there's hardcoding here and there, main reason why I chose that is because we're eventually moving to another CMS and when that happens, we're going to have to refactor most of this anyways and separate everything to a proper SEO model that contains `title`,`description`, `robots` etc.

Reply:
Yep, I'm expecting a major refactor of naming and whole structure as is. Currently it's confusing as heck navigating forestry, since different page types have different implementations 🤯